Makefile

```diff
@@ -20,6 +20,7 @@ core:
 	install -m 0644 -t "$(DESTDIR)$(MANDIR)/man7" -D docs/man/dist/man7/*.7
 	install -m 0644 -t "$(DESTDIR)$(MANDIR)/man8" -D docs/man/dist/man8/*.8
 	install -m 0755 -t "$(DESTDIR)$(EXAMPLES)" -D contrib/*
+	install -m 0755 -t "$(DESTDIR)$(EXAMPLES)" -D examples/*
```

Comment:
With this change, it might be nice to put `contrib/*` in `$(DESTDIR)$(EXAMPLES)/hooks` or something.